doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3407 @@
+{
```

Comment:
<p>This is equivalent to..., but the user doesn't need to remember to do it themselves, and the code stays more readable (or something like that, just find the original a bit clumsy)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>

Reply:
<p>I like the suggestion</p>